src/controllers/coursesController.js

```diff
@@ -1,32 +1,42 @@
 const createError = require('http-errors');
 const expressify = require('expressify')();
-const logger = require('../utils/logger.js');
+// const logger = require('../utils/logger.js');
 const coursesService = require('../services/coursesService');
 
 /**
  * Get courses.
  */
 const getCourses = async (req, res) => {
-  const { page, limit } = req.query;
+  let { page, limit } = req.query;
+  const { authorization } = req.headers;
```

Comment:
not necessary, you will do it in the `authMiddleware`

Reply:
done!